data/json/items/armor.json

```
@@ -10210,5 +10210,112 @@
     "material_thickness": 1,
     "environmental_protection": 1,
     "flags": [ "OVERSIZE", "OUTER", "WATERPROOF", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "bigback",
+    "type": "ARMOR",
+    "name": "high-volume rucksack",
+    "description": "Also know as a \"bigpack\", this rucksack is prized by pack rats but may lead to a bad back.",
+    "weight": 3539,
+    "volume": "21 L",
+    "price": 20000,
+    "material": [ "steel", "leather" ],
+    "symbol": "[",
+    "color": "black",
+    "covers": [ "TORSO" ],
+    "coverage": 95,
+    "encumbrance": 100,
+    "storage": 400,
+    "warmth": 5,
+    "material_thickness": 6,
+    "flags": [ "BELTED", "WATERPROOF", "ONLY_ONE", "OVERSIZE" ]
+  },
+  {
+    "id": "backpack_hiking",
+    "type": "ARMOR",
+    "name": "hiking backpack",
+    "description": "A large sized hiking backpack with plenty of storage space.",
+    "weight": 2291,
+    "volume": "14 L",
+    "price": 10000,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "coverage": 75,
+    "encumbrance": 55,
+    "storage": 220,
+    "warmth": 6,
+    "material_thickness": 6,
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Sheath blade",
+      "holster_msg": "You sheath your %s",
+      "min_volume": 1,
```

```suggestion
      "min_volume": "250ml",
```
I'm honestly not sure if min_volume and max_volume accept the milliliter/liter format.